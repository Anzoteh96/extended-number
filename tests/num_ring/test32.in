715303990269844331523338198755916905259165444910654730459717647488201764074302279420504699920431
-222811643475971294358246559367140559245191453432965877960086624455509686659965248761606424687132
520736844961096967050390099465424333990038938506919269996069351214847751499038177065315832985922
