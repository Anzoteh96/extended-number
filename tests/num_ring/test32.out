Enter the number base b:
Enter two numbers n and m:
b is 715303990269844331523338198755916905259165444910654730459717647488201764074302279420504699920431
n mod b is 492492346793873037165091639388776346013973991477688852499631023032692077414337030658898275233299
m mod b is 520736844961096967050390099465424333990038938506919269996069351214847751499038177065315832985922
n + m is 297925201485125672692143540098283774744847485073953392035982726759338064839072928303709408298790
n - m is 687059492102620401638039738679268917283100497881424312963279319306046089989601133014087142167808
n * m is 198313629988834701964789231459927272993678916579735463366824559822431238590987220940018723640318
n ** m is 665741318387751781919133025232475298164259330040735341116171648190858153438546447563666011353048
